lib/canvas/live_events.rb

```diff
@@ -93,6 +93,19 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
```

Comment:
I assume we're only going to use this id and then fetch the actual file information through the REST API, so there would be no use in adding the `folder_id` to this?